pepdbagent/utils.py

```diff
@@ -104,3 +105,53 @@ def convert_date_string_to_date(date_string: str) -> datetime.datetime:
     :return: datetime format
     """
     return datetime.datetime.strptime(date_string, "%Y/%m/%d") + datetime.timedelta(days=1)
+
+
+def order_samples(results: dict) -> List[dict]:
```

Comment:
I trust this works, is this something worth optimizing in a lower-level if it needs to really be fast?

Reply:
yes, if we want it to be super fast, we should write it in Rust